tests/acceptance/in-option-generate-test.js

```diff
@@ -94,4 +94,19 @@ describe('Acceptance: ember generate with --in option', function() {
         })
       );
   });
+
+  it('generate blueprint foo using sibling path', function() {
+    // build an app with an in-repo addon in a non-standard path
+    return initApp()
+      .then(() => fs.mkdirp('../sibling'))
+      .then(() => generateUtils.inRepoAddon('../sibling'))
```

Comment:
Can you double check what this does? I want to make sure that it _does not_ add `../sibling` to the current projects `package.json`’s `ember-addon.paths`. In other words, the `../sibling` dir should have **no relationship** with the current project...

Reply:
Good call. Fixing.